pkg/kubeconfig/create_kubecfg.go

```diff
@@ -85,56 +85,57 @@ func BuildKubecfg(cluster *kops.Cluster, keyStore fi.Keystore, secretStore fi.Se
 	b := NewKubeconfigBuilder(configAccess)
 
 	b.Context = clusterName
+	b.Server = server
 
-	// add the CA Cert to the kubeconfig only if we didn't specify a SSL cert for the LB
-	if cluster.Spec.API == nil || cluster.Spec.API.LoadBalancer == nil || cluster.Spec.API.LoadBalancer.SSLCertificate == "" {
-		cert, _, _, err := keyStore.FindKeypair(fi.CertificateId_CA)
-		if err != nil {
-			return nil, fmt.Errorf("error fetching CA keypair: %v", err)
-		}
-		if cert != nil {
-			b.CACert, err = cert.AsBytes()
+	if admin {
```

Comment:
Why omit the certificate-authority-data when `admin` is false?

Reply:
Yeah. That wasn't intentional. Fixed.